src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,39 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int BOUND = 10;
+    private K[] keys = (K[]) new Object[BOUND];
+    private V[] values = (V[]) new Object[BOUND];
```

Comment:
move initialization of those values to constructor
```suggestion
    private K[] keys;
    private V[] values;
```